project_national/sdr_upgrades_tmy3.yml

```diff
@@ -43,15 +43,15 @@ postprocessing:
     - County
   publish_annual_results: true
   # AWS upload bucket and athena setup (uncomment during production and testing runs)
-  # aws:
-  #  region_name: us-west-2
-  #  s3:
-  #    bucket: sdr_resstock
-  #    prefix: production_runs
-  #  athena:
-  #    glue_service_role: service-role/AWSGlueServiceRole-default
-  #    database_name: sdr_resstock
-  #    max_crawling_time: 1200
+  aws:
+   region_name: us-west-2
+   s3:
+     bucket: resstock-core
+     prefix: resstock_ci
+   athena:
+     glue_service_role: service-role/AWSGlueServiceRole-default
+     database_name: resstock_core
+     max_crawling_time: 1200
```

Comment:
Are these lines uncommented just for testing?

Reply:
No - they will remain uncommented. I have a script that will take them out in CI when not needed

Reply:
I see. I read the description above in `remove_aws_from_yaml.py` - makes sense.